template/ios/.xcode.env

```diff
@@ -0,0 +1,4 @@
+# PATH to the node executable. You can use the `command -v node` to check what value make sense for your set up
+# This `.xcode.env` file is versioned. To customize your local environment, you can create an `.xcode.local.env`
+# file that is not versioned.
+export NODE_BINARY=/usr/local/bin/node
```

Comment:
Shouldn't we just use `command -v node` here?

Reply:
I think that we would like to avoid a long list of commented examples that users can uncomment. 

What happens when something changes in one of these node managers and the commented code does not work anymore? We have to maintain it. 
Encouraging users to add their favourite `node` manager also defeats the purpose of having a versioned `.env` file that can be different project by project. At that point, we could have kept the old `find-node` buried in the build steps. 

I think that it's ok to add one example, to show the developers how to configure a different node manager, perhaps one with a complex setup, but we should not accept PRs where people add different managers.

Reply:
true. ok lets keep at least one example, maybe a tiny explanation of what should go there, and/or maybe a link to some doc?

i think a doc could easily have a bunch of examples that everyone can make a PR to add/update an example with what they have found, so others can just copy/paste?

Reply:
@pvinis I added some documentation within the `.xcode.env` file itself and I'm going to explain it also in the website. Do you think that it is enough documentation? Do you have something else in mind?